### ISSUE TITLE
Resolve 404s caused by requesting data from invalid endpoints

### ISSUE THREAD

**Issue by simonsinclair** [bbc/simorgh#7944]
### Is your feature request related to a problem? Please describe.
Simorgh is triggering 404's, because it is requesting data from the following _invalid_ endpoints:
```
/ukchina/sty-secondary-column
/ukchina/mostread
/ukchina/mostwatched
```

### Describe the solution you'd like
We should only ever be requesting data for service variants. E.G.:
`/ukchina/sty-secondary-column/trad.json` not `/ukchina/sty-secondary-column`.

### Describe alternatives you've considered
n/a

### Testing notes
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [x] This feature is expected to need manual testing.

### Additional context
None.

**Comment by karinathomasbbc** [comment]
I suspect this might now be resolved by #8186. Let's monitor Sumologic when this goes live on Monday, to see if it has fixed it.

**Comment by karinathomasbbc** [comment]
As suspected, these errors are no longer appearing on the logs, since #8186 was released at approx 9am on Mon 26th October. This issue is now closed.
